HTML/QuickForm.php

```diff
@@ -31,6 +31,10 @@
  * Base class for all HTML classes
  */
 require_once 'HTML/Common.php';
+/**
+ * Static utility methods
+ */
+require_once 'HTML/QuickForm/utils.php';
```

Comment:
Since most of this is non-change I'm just gonna add comments on the lines that are code changes in the first pass & analyse on the second 

Change 1